src/chess/Game.java

```diff
@@ -28,6 +28,8 @@ public void play() {
       System.out.println(board);
       String userInput = InputController.getUserInput("Enter UCI (type 'help' for help): ");
       switch (Command.parse(userInput)) {
+        case BOARD:
+          System.out.println(board);
```

Comment:
good work:+1:

but if once you implement `BOARD` command, what would happen when you type `board`?
The chessboard will be printed out twice, right?
Is this what you expected?
Then, you may consider removing the other printing on line 28 currently.

Reply:
This is just recommendation for now